.github/workflows/ci.yml

```
@@ -91,8 +203,7 @@ jobs:
           npm install -g ganache-cli ilp-settlement-xrp conventional-changelog-cli
```

```suggestion
          npm install -g ganache-cli ilp-settlement-xrp
```

This hit my eye, `rg conventional-changelog` doesn't bring up any matches.

Could create a ci cleanups issue though I'm sure this PR already covers the most.

---

I'll remove this then 👍